.github/workflows/visual.yml

```
@@ -150,44 +149,86 @@ jobs:
             core.setOutput("ready", String(hasCompleteScreenshots));
 
+      - name: Create GitHub App token
+        id: app-token
+        if: >
+          steps.gate.outputs.is_latest == 'true' &&
+          steps.check-artifacts.outputs.ready == 'true' &&
+          (
+            inputs.event != 'pull_request' ||
+            github.event.pull_request.head.repo.full_name == github.repository
+          )
+        uses: actions/create-github-app-token@v3
```

actionlint (1.7.12)

[error] missing input "app-id" which is required by action "actions/create-github-app-token@v3". all required inputs are "app-id", "private-key"

(action)